latex/Research_Strategy.tex

```diff
@@ -450,7 +497,7 @@
 As in SAMPL3~\cite{muddana_sampl3_2012}, these acyclic CB[$n$]-type receptors introduce conformational complexity, and water interactions play a key role.
 Moreover, the presence of 4 anionic groups near the cavity will likely impact the balance between ion-dipole interactions and solvation of the free host.
 
-{\bf Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
+\textbf{Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
```

Comment:
Are we going with "Gibb deep cavity cavitands", "GDCC", or "octa-acid hosts" throughout? We should be consistent in how we briefly refer to these. Even I get confused all the time!